src/lib/mergeThemes.ts

```diff
@@ -64,9 +65,10 @@ export const mergeComponentVariables = (
     return (...args) => {
       const accumulatedVariables = acc(...args)
       const computedComponentVariables = callable(next)(...args)
+      const mergedVariables: ObjectOf<any> = {}
 
-      const mergedVariables = {}
-      _.mapKeys(computedComponentVariables, (variableToMerge, variableName) => {
+      Object.keys(computedComponentVariables).forEach(variableName => {
```

Comment:
`Object.keys` will throw `Uncaught TypeError: Cannot convert undefined or null to object` if called with `null` or `undefined`.  Suggest sticking with lodash methods as they gracefully handle these cases.

Reply:
done, thanks